documentation/release_notes.markdown

```diff
@@ -14,6 +14,29 @@ canonical: "/puppetdb/latest/release_notes.html"
 [metrics]: ./api/metrics/v1/changes-from-puppetdb-v3.html
 [pqltutorial]: ./api/query/tutorial-pql.html
 
+
+4.2.1
+-----
+PuppetDB 4.2.1 is a minor bugfix release.
+that fixes a regression one-armed and
+and or clauses in the query language, as well as a bug that prevented the
```

Comment:
"regression **of** one-armed" ?
Also maybe put "and" and "or" in quotes bc that sentence was kind of hard to parse.